src/ServiceControl.AcceptanceTests/TestSupport/ServiceControlComponentRunner.cs

```diff
@@ -156,6 +156,7 @@ async Task InitializeServiceControl(ScenarioContext context)
                 bootstrapper = new Bootstrapper(settings, configuration, loggingSettings, builder =>
                 {
                     builder.RegisterType<FailedErrorsController>().FindConstructorsWith(t => t.GetTypeInfo().DeclaredConstructors.ToArray());
+                    builder.RegisterType<FailedMessageRetriesController>().FindConstructorsWith(t => t.GetTypeInfo().DeclaredConstructors.ToArray());
```

Comment:
If we're going to keep following this pattern, we can replace it with a generic catch-all registration that will pick up all testing controllers:

```csharp
builder.RegisterAssemblyTypes(typeof(FailedErrorsController).Assembly)
  .AssignableTo<ApiController>()
  .FindConstructorsWith(t => t.GetTypeInfo().DeclaredConstructors.ToArray())
  .AsSelf();
```

That way the next test that needs to do it can just add the controller and it will just work.

Reply:
This is a nice addition. How about a dedicated PR?

Reply:
Added

Reply:
I double-checked and the other component runners don't need this code so we should be fine